cypress-tests/cypress/e2e/ConnectorUtils/Trustpay.js

```diff
@@ -14,6 +14,58 @@ const successfulThreeDSTestCardDetails = {
     "card_cvc": "737",
 };
 
+const idealBankRedirectDetails = {
+    "bank_redirect": {
+        "ideal": {
+            "billing_details": {
+                "billing_name": "John Doe"
+            },
+            "bank_name": "ing",
+            "preferred_language": "en",
+            "country": "NL"
+        }
+    }
+}
+
+const giropayBankRedirectDetails = {
+    "bank_redirect": {
+        "giropay": {
+            "billing_details": {
+                "billing_name": "John Doe"
+            },
+            "bank_name": "ing",
```

Comment:
these two fields are in giropay pm data - `bank_account_bic`, `bank_account_iban`, why is this not added?
if trustpay doesn't require this, please pass it as `none`